lib/tests/streamlit/elements/data_editor_test.py

```diff
@@ -57,9 +59,79 @@ def _get_arrow_schema(df: pd.DataFrame) -> pa.Schema:
 
 
 class DataEditorUtilTest(unittest.TestCase):
-    def test_parse_value(self):
-        # TODO: test parse_value
-        pass
+    @parameterized.expand(
+        [
+            (None, ColumnDataKind.STRING, None),
+            ("hello", ColumnDataKind.STRING, "hello"),
```

Comment:
Is there a reason why float and list is maybe not considered here for string?

Reply:
Added flaot 👍 , list isn't supported yet for editing.. so isn't required here for testing